utoipa-gen/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Changed
 
+* scripts/test.sh: Fix `auto_into_responses` feature declaration (https://github.com/juhaku/utoipa/pull/1252)
```

Comment:
Otherwise great, but should be under new title `Unreleased` and subtitle `Changed` at the top before `5.3.0`

Reply:
oops, I missed that this was for a released version 😅